<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaasInfoWidget</class>
 <widget class="QWidget" name="SaasInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>518</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="widgetLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="placeholderPage">
      <layout class="QGridLayout" name="placeholderPageLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <spacer name="placeholderTopSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>187</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="placeholderLeftSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>222</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="placeholderContentsWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="placeholderContentsLayout">
          <property name="spacing">
           <number>16</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="placeholderIconLabel">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::NoTextInteraction</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="placeholderMessageLayout">
            <property name="spacing">
             <number>8</number>
            </property>
            <item>
             <widget class="QLabel" name="placeholderMessageCaptionLabel">
              <property name="text">
               <string>No services found</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::NoTextInteraction</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="placeholderMessageLabel">
              <property name="text">
               <string>Contact your Channel Partner to add services</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::NoTextInteraction</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="channelPartnerContactButtonLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="channelPartnerContactButtonLeftSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="channelPartnerContactButton">
              <property name="text">
               <string>Channel Partner Contact</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="channelPartnerContactButtonRightSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="placeholderRightSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>221</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="placeholderBottomSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>187</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="servicesUsagePage">
      <layout class="QVBoxLayout" name="servicesUsagePageLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="nx::vms::client::desktop::Panel" name="servicesHeaderWidget" native="true">
         <layout class="QVBoxLayout" name="servicesHeaderLayout">
          <item>
           <widget class="QLabel" name="saasStateCaptionLabel">
            <property name="text">
             <string>Services state</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="saasStateValueLabel"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="horizontalLine">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="servicesContentsWidget" native="true">
         <layout class="QVBoxLayout" name="servicesContentsLayout">
          <item>
           <widget class="QGroupBox" name="servicesUsageGroupBox">
            <property name="title">
             <string>Services Usage</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="servicesUsageGroupBoxLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTreeView" name="servicesUsageItemView">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="showDropIndicator" stdset="0">
                <bool>false</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <property name="rootIsDecorated">
                <bool>false</bool>
               </property>
               <property name="itemsExpandable">
                <bool>false</bool>
               </property>
               <property name="expandsOnDoubleClick">
                <bool>false</bool>
               </property>
               <attribute name="headerStretchLastSection">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::Panel</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/panel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>gridSelectionChanged()</slot>
 </slots>
</ui>
